Lib/test/test_ntpath.py

```diff
@@ -1095,6 +1095,11 @@ def test_isfile_driveletter(self):
             raise unittest.SkipTest('SystemDrive is not defined or malformed')
         self.assertFalse(os.path.isfile('\\\\.\\' + drive))
 
+    @unittest.skipUnless(hasattr(os, 'pipe'), "need os.pipe()")
+    def test_isfile_pipe(self):
+        pr, _ = os.pipe()
+        self.assertFalse(ntpath.isfile(pr))
```

Comment:
```suggestion
        pr, pw = os.pipe()
        try:
            self.assertFalse(ntpath.isfile(pr))
        finally:
            os.close(pr)
            os.close(pw)
```

Reply:
Thanks, I was already wondering if I needed to close these.